_posts/builtin/2022-02-23-container-orchestration-provider.md

```diff
@@ -65,7 +65,13 @@ Containers may be configured using the following fields:
 - **Environment Variables (optional)** - This field accepts a comma-separated list of environment variables, which will be set inside the container when spun up.
 
 - **Entrypoint Override (optional)** - This field accepts a comma-separated list which is used to override the command used to start a container. Example: ```./test.sh,-v,-d,--human-readable```.
-  
+
+- **Memory (optional)** - This field allows the configuration of the maximum amount of memory the container can use in bytes. The value is a positive integer, optionally followed by a suffix of b, k, m, g, to indicate bytes, kilobytes, megabytes, or gigabytes. The minimum and default values depends by the native container orchestrator. If left empty, the memory assigned to the container will be set to a default value.
+
+ - **CPUs (optional)** - This value specifies how many CPUs a container can use. Decimal values are allowed, so if set to 1.5, the container will use at most one and a half cpu resource.
+
+- **GPUs (optional)** - This field configures how many Nvidia GPUs a container can use. Allowed values are 'all' or an integer number.
```

Comment:
Would adding the links to the Docker documentation make sense or the implementation in this PR differs so much that we would make things more confusing?

Reply:
I prefer not to link the Docker documentation, since the orchestrator is a generic provider. Moreover, you are true, it a bit confusing...